### PR TITLE
Correct the privacy group generation. 

### PR DESCRIPTION
**Pull request by Puneetha17**
Update to generate correct privacy group to keep in sync with `Pantheon` and `Orion` generated privacy group.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -61,25 +63,38 @@ function EEAClient(web3, chainId) {
...
-    });
+    const rlp = RLP.encode(participants);
 
-    const rlp = RLP.encode(map).toString("hex");
+    const privacyGroupIdHash = keccak256(rlp).toString("hex");
 
-    const privacyGroupId = keccak256(`0x${rlp}`).toString("hex");
+    const privacyGroup = `0x${Buffer.from(
+      Buffer.from(privacyGroupIdHash, "hex").toString("base64")
+    ).toString("hex")}`;
```
Comment (antonydenyer) [inline src/index.js]:
``` const privacyGroup = Buffer.from(
      keccak256(rlp).toString("base64")
    ).toString("hex"); ```

**Comment by antonydenyer** [comment]
Can you cut this PR down so that it focuses on solving the `eea_getTransactionCount` implementation only?

**Comment by antonydenyer** [comment]
Also, can you ensure that  everything continues to work as expected as per: https://docs.pantheon.pegasys.tech/en/latest/Privacy/Creating-Sending-Private-Transactions.

**Comment by Puneetha17** [comment]
Updated according to the above comments. Verified the multinode example with `pantheon-quickstart`- working fine. 

An issue to consider - For all nodes in `pantheon-quickstart` we do not set the `min-gas-price` to 0 and hence we need to mention a gas price in `index.js`. But according to https://docs.pantheon.pegasys.tech/en/stable/Privacy/Configuring-Privacy/, all the pantheon nodes are started with `--min-gas-price=0` so there will be a conflict when executing the above. 

Which one of the floowing should be the way to go:
- change the `pantheon-quickstart` and so undo the `GAS_PRICE` change
- Update the docs to not specify the flag `--min-gas-price=0`

**Comment by antonydenyer** [comment]
Update the quickstart to run with a `--min-gas-price=0`

File: src/index.js [hunk 2]
```diff
@@ -58,18 +62,60 @@ function EEAClient(web3, chainId) {
...
+      return parseInt(result.data.result, 16);
+    });
+  };
+
   // eslint-disable-next-line no-param-reassign
   web3.eea = {
+    getTransactionCount,
     sendRawTransaction: options => {
       const tx = new PrivateTransaction();
       const privateKeyBuffer = Buffer.from(options.privateKey, "hex");
```
Comment (antonydenyer) [inline src/index.js]: That's hex, not a buffer

File: src/index.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
 const axios = require("axios");
-const PrivateTransaction = require("./privateTransaction");
+const RLP = require("rlp");
+const _ = require("lodash");
+const { keccak256 } = require("./custom-ethjs-util");
 const ethUtil = require("./custom-ethjs-util");
```
Comment (antonydenyer) [inline src/index.js]: duplicate require `const { keccak256, privateToAddress } = require("./custom-ethjs-util");`

**Pull request merged**